src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -3779,7 +3807,7 @@ public int forceRepairAsync(String keyspace, RepairOption options, boolean legac
         if (!inProgressCommand.isPresent()) {
             new Thread(createRepairTask(cmd, arguments, legacy)).start();
         } else {
-            logger.info("Combining new repair request with in-progress (identical) repair command #{}", cmd);
+            logger.info("Combining new repair request with in-progress (identical) repair command #{}", UnsafeArg.of("command", cmd));
```

Comment:
cmd is an int, so this is probably safe, right?

Reply:
Fair enough... didn't check the type, assumed this was the literal command.